src/Microsoft.AspNetCore.Hosting/WebHostBuilder.cs

```diff
@@ -224,7 +225,7 @@ private class WrappedApplicationEnvironment : IApplicationEnvironment
         {
             public WrappedApplicationEnvironment(string applicationBasePath, IApplicationEnvironment env)
             {
-                ApplicationBasePath = applicationBasePath;
+                ApplicationBasePath = Path.GetFullPath(applicationBasePath);
```

Comment:
When we talked the idea was to resolve relative to the .exe but for a deployed app isn't it more useful/natural to resolve it relative to a current directory?